static-site/src/components/Footer/index.jsx

```diff
@@ -55,6 +56,12 @@ class Footer extends React.Component {
   render() {
     return (
```

Comment:
[minor] we typically render a horizontal line above the footer but don't do this on the main splash page. With the changes introduced here I think it'd be clarifying to also render the horizontal line on the main splash page

Reply:
Added: 6b6f4520159c9aaa2bb437e41c22c4d08162d25d